lib/_http_client.js

```diff
@@ -327,23 +327,33 @@ function socketCloseListener() {
 
   // NOTE: It's important to get parser here, because it could be freed by
   // the `socketOnData`.
-  var parser = socket.parser;
-  if (req.res && req.res.readable) {
+  const parser = socket.parser;
+  const res = req.res;
+  if (res) {
     // Socket closed before we emitted 'end' below.
-    if (!req.res.complete) req.res.emit('aborted');
-    var res = req.res;
-    res.on('end', function() {
+    if (!res.complete) {
+      res.aborted = Date.now();
```

Comment:
Let's just make this `true`.

Reply:
@apapirovski: I've got a separate PR for that #20230

Reply:
Yeah, I saw that but I don't think this should land with it being a date. `Date.now()` is pretty slow so any aborted requests would kinda tank performance of the client. Perhaps let's leave it out for now and just have it land with the patch that makes it Boolean across the board?

Reply:
Removed.